download-libs.sh

```diff
@@ -42,6 +48,7 @@ if [ ! -d closure-compiler/.git ]; then
   fi
   git clone --depth 1 https://github.com/google/closure-compiler closure-compiler
```

Comment:
Here's the style of git clone that we went with, which lets us still do a depth 1 clone and has a more descriptive name for the version:
git clone --branch maven-release-v20150315 --depth 1 https://github.com/google/closure-compiler closure-compiler
https://github.com/google/end-to-end/commit/75fb0f27c62766a6f8524d743c51a310a9a8b7d3